openshift/ci-operator/build-image/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Dockerfile to bootstrap build and test in openshift-ci
 
-FROM registry.ci.openshift.org/openshift/release:golang-1.18
+FROM registry.ci.openshift.org/openshift/release:golang-1.19
```

Comment:
I think the build image aka base should be 1.19 as well. Otherwise we'll create a version hell for use slowly. :)